starwars/starwars_people/services.py

```diff
@@ -0,0 +1,47 @@
+from abc import ABC
+
+import petl
+
+from starwars_people.adapters import StarWarsAdapters
+from starwars_people.clients import StarWarsApiClient
+from starwars_people.models import Dataset
+from starwars_people.utils import generate_unique_name
+from django.core.files import File
+import os
+
+
+class DataWriterDBSaver(ABC):
+    file_name = None
+    client = StarWarsApiClient()
+
+    def write_data_and_save_to_file(self):
+        pass
+
+    def save_file_to_db_and_remove_from_disc(self):
+        pass
+
+
+class CSVDataWriterAndDBSaver(DataWriterDBSaver):
+
+    def __init__(self):
+        self.file_name = generate_unique_name()
+        self.adapter = StarWarsAdapters()
+
+    def write_data_and_save_to_file(self):
+        data = self.adapter.save_all_filtering(people_data=self.client.get_people_data(), planets_data=self.client.get_planets_data())
```

Comment:
move this logic out of this class. You need separate service where you inject your client, adapter, and data writer

Reply:
also, for better readability
`people_data=self.client.get_people_data()
  planets_data=self.client.get_planets_data()
  data = self.adapter.save_all_filtering(people_data=people_data, planets_data=planets_data
`